src/_posts/platform/app/2000-01-01-app-manifest.md

```diff
@@ -110,6 +110,39 @@ The JSON manifest also lets you declare scripts which will be executed as [postd
 }
 ```
 
+## Container Formation
+
+The `scalingo.json` manifest can be used to setup a container formation. A container formation defines any type of container that will be launched on startup.
+
```

Comment:
```suggestion
The `scalingo.json` manifest can be used to setup a container formation. A container formation defines the shape of the containers that will be launched.

```

`type` seems too vague, shape or size is more adequate IMO.
Furthermore, the word `on startup` doesn't seems to add anything, isn't it?

Reply:
Your true :), I commit the suggestion